Sources/Fog/View/FogRngSet.swift

```diff
@@ -25,13 +25,20 @@ final class FogRngSet {
         numOutputs: PositiveInt,
         minOutputsPerSelectedRng: Int
     ) -> FogSearchAttempt {
+        logger.info(
+            "requestedBlockCount: \(redacting: String(describing: requestedBlockCount)), " +
+                "numOutputs: \(redacting: numOutputs.value), " +
+                "minOutputsPerSelectedRnd: \(redacting: minOutputsPerSelectedRng)")
```

Comment:
`minOutputsPerSelectedRng` doesn't need to be redacted. Also typo, should be `minOutputsPerSelectedRng`.